tests/www/test_views.py

```diff
@@ -561,15 +626,15 @@ def test_health(self):
             BaseJob.job_type == 'SchedulerJob',
             BaseJob.state == 'running',
             BaseJob.latest_heartbeat == last_scheduler_heartbeat_for_testing_1,
-        ).delete()
+            ).delete()
```

Comment:
Please avoid making formatting only changes to lines you don't otherwise touch, as it makes the git history messier.

Reply:
sorry @ashb, Looks like my IDE does that automatically.
I will fix these.

Reply:
@pateash No problem, I know how it happens, it just makes my life as a release manager much harder :)